### DISCUSSION TITLE
[Q] Multitrack: disable scroll, enable redraw on resize

### DISCUSSION THREAD

**Discussion by joacoespinosa** [katspaugh/wavesurfer.js#3612] [Q&A] [answered]
The multitrack "region" seems to slightly exceed its containers width (see attached)

I was wondering if there is a way to force it to respect its containers dimensions. I would think that disabling scrolling could be a solution but I can't seem to find such option either.

Also, the tracks are not redrawn on resize, which is a nice wavesurfer behaviour. I see a lot of space when moving the browser from a lower resolution screen to a higher one. Is there a way to enable redraw on resize?

Thank you,

[video.webm](https://github.com/katspaugh/wavesurfer.js/assets/23172865/e2674208-c370-4f02-9bd4-91feb889d664)

**Comment by katspaugh** [comment, marked as answer]
Please see https://github.com/katspaugh/wavesurfer-multitrack#support
